include/irr/asset/IDescriptorSet.h

```diff
@@ -143,9 +143,11 @@ class IEmulatedDescriptorSet
 			if (!_layout)
 				return;
 
-			const auto& bindings = _layout->getBindings();
+			auto bindings = _layout->getBindings();
+            assert(bindings.length());
+            auto lastBnd = (bindings.end()-1);
```

Comment:
std::array has a `back()` method, we should have one too in `dynamic_array`

Reply:
it already has

Reply:
why not use it?

Reply:
`bindings` is `SRange` and not `dynamic_array`